improver_tests/synthetic_data/test_set_up_test_cubes.py

```diff
@@ -839,14 +849,14 @@ def test_error_too_many_dimensions(self):
     def test_error_not_enough_dimensions(self):
         """Test error is raised if 3D input cube and both realizations and heights provided"""
         realizations = [0, 3, 4]
-        height_levels = [1.5, 3.0, 4.5]
+        vertical_levels = [1.5, 3.0, 4.5]
         msg = (
             "Input data must have 3 dimensions to add both realization "
-            "and height coordinates: got 2"
+            "and diagnostic coordinates: got 2"
```

Comment:
```suggestion
            "and vertical coordinates: got 2"
```
Will become this if you implement my suggestions in that function.